packages/core/test/slider/multiSliderTests.tsx

```diff
@@ -215,6 +215,25 @@ describe("<MultiSlider>", () => {
     });
 
     describe("validation", () => {
+        let expectedErrors = 0;
+        let actualErrors = 0;
+        function onError(e: ErrorEvent) {
```

Comment:
I'm curious why these changes were necessary...?

Reply:
because the errors weren't asserted correctly. i took the issue and the implementation from https://github.com/facebook/react/issues/11098#issuecomment-355032539